### ISSUE TITLE
New suggestion

### ISSUE THREAD

**Issue by calderaricaio** [earlduque/Slack-Dark-Theme#10]
Hi Guys, I found a way to add css code to avoid that extra step where the user needs to insert the hexadecimal colors inside user preferences. Is that a desirable thing?
I mean, when I installed the dark theme for the first time I thought it was with some problem because my sidebar was kept same color, so I read the readme again and find out that I had to do another step, but for me as a user it seemed another step to complete the task so, maybe we can add that same hexadecimals as default for the theme so when someone install it it's already 100% ready? What do you guys think? Can I improve it?

**Comment by earlduque** [comment]
In my opinion, generally speaking, if it's settable by the user within the interface we should avoid affecting it.

I'll keep this issue open to see if others have any thoughts on it though.

**Comment by calderaricaio** [comment]
I agree that if the user can change it manually we will be replacing that feature, but at the same time, its another extra step. Also when I first installed the theme it gave me a wrong impression that it wasn't working correctly. Another point of view is that, if I took time to add a dark theme on slack I really want it to be entirely black, so why do I need to change myself the other colors anyway? So i mean either I want it black or not, so it's more like a feature we gonna keep alive but doesn't need to be once all the thing is already done for you as is... otherwise why using the theme at all?
